fenix/app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkController.kt

```diff
@@ -195,8 +195,6 @@ class DefaultBookmarkController(
                         startLoading = load,
                     )
                 }
-                activity.browsingModeManager.mode =
-                    BrowsingMode.fromBoolean(mode == BrowsingMode.Private)
```

Comment:
Is the browsing mode change here handled anywhere? You had comments elsewhere where we have remove this redundancy, but it's not obvious to me in this file.

Reply:
Same answer here as in the [below comment](https://github.com/mozilla-mobile/firefox-android/pull/4646#discussion_r1442251706)